src/librustdoc/clean/types.rs

```diff
@@ -373,8 +373,8 @@ impl Item {
         self.def_id.as_def_id().and_then(|did| tcx.lookup_stability(did))
     }
 
-    crate fn const_stability<'tcx>(&self, tcx: TyCtxt<'tcx>) -> Option<&'tcx ConstStability> {
-        self.def_id.as_def_id().and_then(|did| tcx.lookup_const_stability(did))
+    crate fn const_stability<'tcx>(&self, tcx: TyCtxt<'tcx>) -> Option<ConstStability> {
```

Comment:
Do you want me to do the same for `stability`? (the method before this one.)

Reply:
Hmm, I didn't realize the TyCtxt methods return `&'tcx {Const}Stability`. Maybe it'd be best to revert the change to `const_stability()` and instead open a separate PR that changes the TyCtxt methods? I'm not sure why they return references.

Reply:
The query engine dark magic. :3

Reply:
Hehe. Let's leave `stability` alone for now since it'd be a much bigger change. You can keep the `const_stability` change though.

Reply:
Ok, will be interesting to see if we can change the query engine afterwards.